src/scss/components/statistics.scss

```diff
@@ -0,0 +1,125 @@
+.statistics {
+  position: relative;
+  display: block;
+  z-index: 1;
+  overflow: hidden;
+
+  &__container {
+    padding-top: 12rem;
+    padding-bottom: 12rem;
+    background-color: var(--brand-navy);
+
+    @include breakpoint(md) {
+      padding-top: 15.8rem;
+      padding-bottom: 15.8rem;
+    }
+  }
+
+  &__headlines {
+    @include grid;
+  }
+
+  &__eyebrow {
+    grid-column: 1/9;
+    justify-content: center;
+    display: flex;
+    color: var(--white);
+    font-size: 1.6rem;
+    font-weight: var(--font-bold);
+    letter-spacing: 0.2em;
+    line-height: 1.6rem;
+    margin: 0;
+
+    @include breakpoint(md) {
+      grid-column: 4/10;
+    }
+  }
+
+  &__title {
+    @include heading-3;
+
+    grid-column: 1/9;
+    justify-content: center;
+    display: flex;
+    text-align: center;
+    color: var(--white);
+    padding-top: 1rem;
+    margin: 0;
+
+    @include breakpoint(md) {
+      grid-column: 4/10;
+      padding-top: 0.8rem;
+    }
+  }
+
+  &__items {
+    @include grid;
+
+    grid-template-columns: 1fr;
+
+    @include breakpoint(md) {
+      grid-template-columns: repeat(4, 1fr);
+    }
+  }
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 6.4rem;
+    margin: 0;
+
+    @include breakpoint(md) {
+      padding-top: 8.8rem;
+    }
+  }
+
+  &__value {
+    font-size: 80px;
```

Comment:
Great job!, change this px to rem  :)

Reply:
Thanks teacher for your appreciation! I deleted it because the real font-size is the last rule that I included: 12rem :) !